src/wymeditor/editor/undo-redo.js

```diff
@@ -79,6 +79,9 @@ WYMeditor.UndoRedo.prototype._do = function (what) {
         if (history.changesetsBack.length === 0) {
             return;
         }
+        if (!history.changesetsFore.length) {
+            undoRedo._add()
```

Comment:
Yup. That was my mistake, exactly. Sorry about that.

This still needs a semicolon, though.